kernel/src/main/scala/cats/kernel/instances/JavaTimeInstances.scala

```diff
@@ -0,0 +1,55 @@
+package cats.kernel.instances
+
+import cats.kernel.{CommutativeGroup, Eq, Hash, Order}
+import java.time._
+
+trait JavaTimeInstances {
+
+  implicit final val catsKernelStdOrderForduration
+    : Hash[Duration] with Order[Duration] with CommutativeGroup[Duration] =
```

Comment:
we aren't testing the commutative group are we? I don't see where those laws are checked.

Reply:
The laws seem to be failing. I am removing that instance until that is fixed.

Reply:
So, the problem I am having with some of the tests is that, if it generates very large values, it is causing overflows. Not sure how we should address this problem?

Reply:
I was able to fix it by defining a generator for the `Duration` class itself. It is just copied from the one for `FiniteDuration`.